libfwupdplugin/fu-bluez-device.c

```diff
@@ -107,14 +185,14 @@ fu_bluez_device_to_string (FuDevice *device, guint idt, GString *str)
 	FuBluezDevice *self = FU_BLUEZ_DEVICE (device);
 	FuBluezDevicePrivate *priv = GET_PRIVATE (self);
 
-	if (priv->uuid_paths != NULL) {
+	if (priv->uuids != NULL) {
 		GHashTableIter iter;
 		gpointer key, value;
-		g_hash_table_iter_init (&iter, priv->uuid_paths);
+		g_hash_table_iter_init (&iter, priv->uuids);
 		while (g_hash_table_iter_next (&iter, &key, &value)) {
```

Comment:
I think adding `const FuBluezUuid *uuid = (const FuBluezUuid *) value;` here will be more useful for the future.

Reply:
maybe drop the consts too. Not sure.